tests/queries/0_stateless/01818_input_format_with_names_use_header.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+${CLICKHOUSE_CLIENT_BINARY} -q "DROP TABLE IF EXISTS 01818_with_names;"
```

Comment:
Did you mean `${CLICKHOUSE_CLIENT}`?

Reply:
Yes, I copy-pasted this from another test, sorry.